src/bin/cql-stress-cassandra-stress/java_generate/distribution/enumerated.rs

```diff
@@ -0,0 +1,36 @@
+use anyhow::Result;
+use rand_distr::{Distribution, WeightedIndex};
+
+#[derive(Clone)]
+pub struct EnumeratedDistribution<T: Copy + std::fmt::Display> {
```

Comment:
As a principle, trait bounds should be avoided as possible in struct/enum definitions, unless they are required for the struct/enum definition to compile. https://stackoverflow.com/a/66369912